docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -0,0 +1,862 @@
+| true
+
+| `HTTP_RATE_LIMIT_ENABLE`
+| string
+| To enable the HTTP rate limit, this environment variable is required to be `true`.
+| false
+
+| `HTTP_RATE_LIMIT_CONNECTIONS`
+| number
+| Defines the maximum number of HTTP requests during a period.
+Defaults to 200 requests per window.
+| 200
+
+| `HTTP_RATE_LIMIT_WINDOW`
+| number
+| Defines the time for which a record of requests should be kept in the memory (in seconds).
+The default duration of a window is 10 seconds.
+| 10
+
+| `ENABLE_HTTP_CACHE_CONTROL`
+| string
+| To enable response caching, this environment variable is required to be `true`.
+This would include the Cache-Control header within the responses.
+| true
+
+| `HTTP_CACHE_CONTROL_DIRECTIVES`
+| string
+| The `Cache-Control` HTTP directive can be overridden with this environment variable.
+| public, max-age=10
+
+|===
+
+--
+Sample::
++
+--
+[source,js]
+----
+module.exports = {
+  apps: [
+    // Other types of configurations.
+    {
+      name: "lisk-service-gateway",
+      // [...]
+      env: {
+        PORT: "9901",
+        HOST: "127.0.0.1",
+        // --- Remember to set the properties below
+        SERVICE_BROKER: "redis://localhost:6379/0",
+        SERVICE_GATEWAY_REDIS_VOLATILE: "redis://localhost:6379/5",
+        ENABLE_HTTP_API: "http-status,http-version3,http-exports",
+        ENABLE_WS_API: "blockchain,rpc-v3",
+        GATEWAY_DEPENDENCIES: "indexer,connector",
+        WS_RATE_LIMIT_ENABLE: "false",
+        WS_RATE_LIMIT_CONNECTIONS: 5,
+        WS_RATE_LIMIT_DURATION: 1, // in seconds
+        ENABLE_REQUEST_CACHING: "true",
+        JSON_RPC_STRICT_MODE: "false",
+        HTTP_RATE_LIMIT_ENABLE: "false",
+        HTTP_RATE_LIMIT_CONNECTIONS: 200,
+        HTTP_RATE_LIMIT_WINDOW: 10, // in seconds
+        HTTP_CACHE_CONTROL_DIRECTIVES: "public, max-age=10",
+        ENABLE_HTTP_CACHE_CONTROL: "true",
+      },
+    },
+    // Other types of configurations.
+  ],
+};
+----
+--
+=====
+
+
+=== Connector
+
+[tabs]
+=====
+Properties::
++
+--
+[cols="2,1,3,3",options="header",stripes="hover"]
+|===
+| Environment variable
+|Datatype
+|Description
+|Sample
+
+| `SERVICE_BROKER`
+| string
+| URL of the microservice message broker (NATS or Redis).
+| redis://localhost:6379/0
+ 
+| `LISK_APP_WS`
+| string
+| URL to connect with the Lisk application node over WebSocket.
+| ws://localhost:7887
+
+| `USE_LISK_IPC_CLIENT`
+| string
+| Boolean flag to enable IPC-based connection to the Lisk application node.
+Requires `LISK_APP_DATA_PATH` to be set.
+| true 
+
+| `LISK_APP_DATA_PATH`
+| string
+| Data path to connect with the Lisk application node over IPC.
+| ~/.lisk/lisk-core
+
+| `GEOIP_JSON`
+| string
+| URL of the GeoIP server
+| https://geoip.lisk.com/json
+
+| `GENESIS_BLOCK_URL`
+| string
+| URL of the Lisk SDK-based application's genesis block.
+Only to be used when the genesis block is large enough to be transmitted over API calls within the timeout
```

Comment:
Could we please ensure that all the descriptions end with a `.` for consistency?
```suggestion
Only to be used when the genesis block is large enough to be transmitted over API calls within the timeout.
```